src/codemagic/utilities/argument_utils.py

```diff
@@ -0,0 +1,8 @@
+def get_binary_arguments_value(value, not_value):
```

Comment:
Instead of creating new `argument_utils` module we could just add this functionality to `Argument` class:
```python
class Argument(ArgumentProperties, enum.Enum):

    @classmethod
    def resolve_optional_two_way_switch(cls,
                                        is_switched_on: Optional[bool],
                                        is_switched_off: Optional[bool]) -> Optional[bool]:
        if {is_switched_on, is_switched_off} in ({True}, {False}):
            raise ValueError('Neither of the switches, or exactly one can be truthy at the time')
        elif is_switched_on is True:
            return True
        elif is_switched_off is True:
            return False
        else:
            return None
```

Reply:
added [here](https://github.com/codemagic-ci-cd/cli-tools/pull/67/commits/71f2ca10806aa46be85d0feca84182780939f3d7#diff-46d5d4f67d34f1a5bcd1808beeab3ac0dd2953aa339f92151b8b45b4553b2e31R18)